docs/xblock-tutorial/edx_platform/overrides.rst

```diff
@@ -0,0 +1,75 @@
+.. _Replace a Preinstalled XBlock With a Custom Implementation:
+
+##########################################################
+Replace a Preinstalled XBlock With a Custom Implementation
+##########################################################
+
+In XBlock ``v5.1.0``, the ability was introduced to override an XBlock with a custom
+implementation.
+
+This can be done by:
+
+1. Creating an XBlock in a new or existing Django Plugin, installed into ``edx-platform``.
+
+2. Adding the ``xblock.v1.overrides`` entry point in ``setup.py``, pointing to your
+   custom XBlock.
+
+This works with updated logic in ``load_class``'s ``default_select``, which gives
+load priority to a class with the ``.overrides`` suffix.
+
+This can be disabled by providing a different ``select`` kwarg to ``load_class`` which
+ignores or otherwise changes override logic.
+
+*******
+Example
+*******
+
+Imagine there is an XBlock installed ``edx-platform``:
+
+.. code:: python
+
+    # edx-platform/xblocks/twitch_plays_pokemon.py
+    class TwitchPlaysPokemonBlock(XBlock):
```

Comment:
sorry to be a grouch, but I'm not sure the whimsy meshes with the tone of our existing admin-facing docs 😛 

perhaps, to be maximially concrete, it would be helpful to reference a real edx-platform XBlock, like edx-platform/xmodule/poll_block.py ?